.github/workflows/tests.yml

```
@@ -0,0 +1,30 @@
+# Authored by Alexandra N. Walker <alex.walker@indicio.tech> ☀️
+
+name: Tests
+on:
+  push:
+    branches: 
+      - main
+      - feature/revocation
+      - test/tests
+  pull_request:
+    branches:
+      - main
+      - feature/revocation
+
+env:
+  POETRY_VERSION: 1.1.11
+
+jobs:
+  test:
+  name: Tests
+  runs-on: ubuntu-latest
+  strategy:
+    matrix:
+      python-version: [3.6, 3.7, 3.8, 3.9]
```

```suggestion
      python-version: ["3.7", "3.8", "3.9", "3.10"]
```
However, currently, our pyproject.toml is require 3.9+. Let's change that to be 3.7+.